### ISSUE TITLE
spectrum widget in ms/ms

### ISSUE THREAD

**Issue by chubukov** [ElucidataInc/ElMaven#396]
I have tested a little bit the new feature described in #170, where you can put in a product-precursor into the mz window.

It seems to be working at least to some extent -- I was able to get some EICs extracted. One of the things that has made it very difficult to test is that the interaction with the spectrum widget it not very good. By default it does not seem to be centered on the right mass, and any click inside it (even a right click to do "reset zoom") changes the EIC window.

**Comment by sahil21** [comment]
@chubukov There is some issue with picking up the right scan also #341. We can look into this when we will work on MS2 spectra.

**Comment by chubukov** [comment]
@sahil21 That might be part of it for sure. But I also noticed that the peak mz value was neither the product nor the precursor, but something random.

**Comment by shubhra-agrawal** [comment]
@chubukov The issue with picking up the right scan has been fixed. The remaining issue is display of incorrect prodMz value in the spectra widget title. It shows base peak m/z of the scan which may or may not be the same as the product m/z. But the mass spectra pulled out is correct as validated against Skyline. 
We'll take this up in the next 1-2 sprints.

**Comment by chubukov** [comment]
@shubhra-agrawal cool, glad to hear it. 

You've surely also noticed that in general, the interactivity of the spectrum widget (zoom, unzoom, hover) is not very smooth. I don't want you to spend a ton of time on these UI issues, but if there are some small fixes that can be made, I'm sure they'd be appreciated.

**Comment by shubhra-agrawal** [comment]
@chubukov Yes, the plot does not fit the widget properly either so data points at the extremes are not visible at times. We'll see what we can do about that.
